src/settings.py

```diff
@@ -0,0 +1,11 @@
+from pydantic.v1 import BaseSettings
+
+
+class Settings(BaseSettings):
+    DATABASE_URL: str | None = "sqlite+aiosqlite:///./city_temperature.db"
+
+    class Config:
+        case_sensitive = True
+        enf_file = "../.env"
```

Comment:
There is a typo in the `Config` class: `enf_file` should be `env_file` to correctly specify the path to the environment file.